manuscript/01_introduction.md

```diff
@@ -50,7 +50,11 @@ As an alternative, therapy resistance of subpopulations can be characterized in
 
-Other computational models for cell lineages have yet to incorporate a tHMM that provides a full spectrum of underlying cell states based on phenotypic measurements for every member in a lineage. Tree-based models utilizing linkage and inheritance information  have previously been employed in phylogeny discovery and evolutionary analyses,[@Bykova] and such models have been validated in molecular systems, such as identifying chromatin hidden states by observing histone modifications.[@Kuchen; @Biesinger] None of these models, however, have been developed for cell phenotype analysis with potential application in tumor-specific cancer therapy, nor have any of these models been provided open source for use by the scientific community. We hope to provide phenotypic measurements of cell fitness as a novel methodology for the following bedside and bench applications: (1) analyzing cells in heterogeneous cancer by providing unseen features of the inherent properties of cellular functioning and (2) providing biomedical researchers an analytical tool for understanding cellular heterogeneity within their experiments. In essence, this work will assist researchers, pathologists, oncologists, and the like in distinguishing the structure of heterogeneous cell subpopulations and identifying single-cell response to cancer therapy. 
+We are motivated by the need to better quantify cell drug response by addressing the problem of cell plasticity and heterogeneity. In particular, we want to study the effects of stochastic transitions between individual cell states with low probabilities but have large substantial effects on the resulting heterogenous population's response to drug. Genetic _association_ studies marked by efforts like the Cancer Cell Line Encyclopedia, which use population level samples to find common risk factors with smaller effect sizes, are valuable nut lack the ability to find rare and meaningful transitions. [@barretina2012cancer] In genetic studies, _linkage_ analyses have found success in situations where the risk factors being sought have large effect sizes but are relatively rare. These linkage studies use pairs of affected relatives like siblings to identify genes that are shared more frequently among those who exhibit a known phenotype [@concannon2009genetics]. In the cellular case, linkage analyses rely on lineage trees to identify groups of related cells or subpopulations. In general, hidden Markov models (HMMs) let one infer cell state from indirect measurements. By operating on a lineage tree, tHMMs enable us to perform a single cell linkage analysis of cellular heterogeniety.
```

Comment:
Rare, large effect is one situation where linkage analysis shines, but it is also uniquely powerful when there are multiple molecular changes/routes that give rise to a common phenotypic change. For example, if mutating seven different genes can all give rise separately to disease X, looking in one family allows you to focus on the effect of one gene rather than averaging over them. I believe an example of this was the effect of BRCA2 mutations in breast cancer. I think this might be a more applicable strength of this analysis, because a population of cells may have members who are resistant for differing underlying reasons.